Foundation/Unit.swift

```diff
@@ -20,32 +20,63 @@ public class UnitConverter : NSObject {
      @param value Value in terms of the unit class
      @return Value in terms of the base unit
      */
-    public func baseUnitValue(fromValue value: Double) -> Double { NSUnimplemented() }
+    public func baseUnitValue(fromValue value: Double) -> Double {
+        return value
+    }
     
     
     /*
      This method takes in a value in terms of the base unit of a unit's dimension and returns the equivalent value in terms of the unit.
      @param baseUnitValue Value in terms of the base unit
      @return Value in terms of the unit class
      */
-    public func value(fromBaseUnitValue baseUnitValue: Double) -> Double { NSUnimplemented() }
+    public func value(fromBaseUnitValue baseUnitValue: Double) -> Double {
+        return baseUnitValue
+    }
 }
 
 public class UnitConverterLinear : UnitConverter, NSSecureCoding {
     
     
-    public var coefficient: Double { get { NSUnimplemented() } }
+    public private(set) var coefficient: Double
     
-    public var constant: Double { get { NSUnimplemented() } }
+    public private(set) var constant: Double
```

Comment:
we also need to override `baseUnitValue(fromValue:)` and `value(fromBaseUnitValue:)`. Here is the ObjC implementation:

```objc
- (double)baseUnitValueFromValue:(double)value {
    return (value  * _coefficient) + _constant;
}

- (double)valueFromBaseUnitValue:(double)baseUnitValue {
    return (baseUnitValue + (-1 * _constant)) / _coefficient;
}
```